apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/entity/state/exchange/Exchange.java

```diff
@@ -0,0 +1,61 @@
+/*
+ *  Copyright © 2018-2020 Apollo Foundation
+ */
+
+package com.apollocurrency.aplwallet.apl.core.entity.state.exchange;
```

Comment:
the same as above

Reply:
There will be two classes - exchange + exchange_request, that is why I created package.